addons/zeus/functions/fnc_moduleHeal.sqf

```diff
@@ -43,8 +43,17 @@ switch (false) do {
 };
 
 // Heal validated target
-if (["ace_medical"] call EFUNC(common,isModLoaded) && {EGVAR(medical,level) > 0}) then {
-    [QEGVAR(medical,treatmentAdvanced_fullHealLocal), [_unit, _unit], _unit] call CBA_fnc_targetEvent;
+if (["ace_medical"] call EFUNC(common,isModLoaded)) then {
+    TRACE_1("healing with ace_medical",_unit);
+    if (_unit getVariable [QEGVAR(medical,inCardiacArrest), false]) then {
+        TRACE_1("exiting cardiac arrest",_unit);
+        [QEGVAR(medical,CPRSucceeded), _unit, _unit] call CBA_fnc_targetEvent;
+    };
+    if (_unit getVariable ["ACE_isUnconscious",false]) then {
+        TRACE_1("waking up",_unit); // wake up first or unconc variables will be reset
+        [QEGVAR(medical,WakeUp), _unit, _unit] call CBA_fnc_targetEvent;
+    };
+    [QEGVAR(medical_treatment,treatmentFullHealLocal), [_unit], _unit] call CBA_fnc_targetEvent;
```

Comment:
Wouldn't it be cleaner to have these checks (cardiac arrest and isUnconscious) direclty in `treatmentFullHealLocal`?